Userland/Applications/Presenter/Presentation.h

```diff
@@ -37,6 +39,7 @@ class Presentation {
     void next_frame();
     void previous_frame();
     void go_to_first_slide();
+    void set_color_scheme(Gfx::Color, Gfx::Color);
```

Comment:
Please mention the parameter names here as well; we only leave out parameter names for declarations if their purpose is obvious (e.g. setters).